org.kde.hana.yml

```diff
@@ -0,0 +1,25 @@
+# SPDX-FileCopyrightText: 2025 George Florea Bănuș <georgefb899@gmail.com>
+# SPDX-License-Identifier: CC0-1.0
+app-id: org.kde.hana
+runtime: org.kde.Platform
+runtime-version: 6.10
+sdk: org.kde.Sdk
+command: hana
+rename-icon: hana
+copy-icon: true
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --filesystem=home
```

Comment:
home is essentially sandbox escape. can this be limited to a few commom folders like xdg-pictures? or better go through the filechooser portal?

Reply:
> Wouldn't that require the user to select an output folder each time they use the app?

If it uses a separate folder each time, then yes, but a single folder once selected through the filechooser portal should be persistently made available by Flatpak.

usually these use a single location to save the files.